test/unit/controllers/subscription/VatDetailsEuControllerSpec.scala

```diff
@@ -60,6 +61,18 @@ class VatDetailsEuControllerSpec extends ControllerSpec with Checkers with Befor
     mockCountries
   )
 
+  override lazy val app: Application = new GuiceApplicationBuilder()
```

Comment:
This will go when you change the Countries class, right?

Reply:
Yes, didn't want to mess with this spec and implementation right now